exchanges/kraken/kraken_wrapper.go

```diff
@@ -382,8 +389,18 @@ func (k *Kraken) FetchTradablePairs(ctx context.Context, a asset.Item) (currency
 			if err != nil {
 				return nil, err
 			}
+			limits = append(limits, order.MinMaxLevel{
+				Asset:                  a,
+				Pair:                   pair,
+				PriceStepIncrementSize: info.TickSize,
+				MinimumBaseAmount:      info.OrderMinimum,
+			})
 			pairs = append(pairs, pair)
 		}
+
+		if err := k.LoadLimits(limits); err != nil {
+			log.Errorf(log.ExchangeSys, "%s Error loading exchange limits: %v", k.Name, err)
```

Comment:
Hello! Thank you for adding this.
We have an interface function `UpdateOrderExecutionLimits` which would be a better place for this!
Checkout Binance for their implementation as a guide

`UpdateOrderExecutionLimits` gets called when an exchange is setup, so you wouldn't need to also make the modifications you've done in the `Run` function

Reply:
@gloriousCode 
I agree with implementing the Interface function. We missed that.

However: Are you sure that `UpdateOrderExecutionLimits` gets called when an exchange is setup?
I think that's only true if the exchange wrapper itself does so, like binance wrapper and okx wrapper,
and they actually do it in Run.

The quandary here with kraken's api was not wanting to call fetch symbols twice, because of the delay, performance impact, etc.
I really can't see a nice way around it.
What I think we'll do is keep the implementation that FetchTradingPairs does the load, but then move the else statement to call UpdateOrderExecutionLimits and just have that call Fetch Pairs.
Best compromise, implements the interface but doesn't introduce extra load if autoUpdate is on.

Reply:
Yeah I misread the code with the calling of `UpdateOrderExecutionLimits`, it does get called in `Run`. What you describe as fixes should be good 🚀

Reply:
that's done

Reply:
Thanks for that. There's a linter issue still for a blank line in kraken_test.go.

A test in our exchange_wrapper_standards testing suite is failing because OrderLimits does not expect a partial implementation. I'll create a PR to add `CheckOrderExecutionLimits` to the ignore list

Reply:
the whitespace is now fixed